feature/quicksettings/src/main/java/com/google/jetpackcamera/feature/quicksettings/ui/QuickSettingsComponents.kt

```diff
@@ -118,7 +120,15 @@ fun QuickSetFlash(
         FlashMode.ON -> CameraFlashMode.ON
     }
     QuickSettingUiItem(
-        modifier = modifier,
+        modifier = modifier
+            .semantics {
+                contentDescription =
+                    when (enum) {
+                        CameraFlashMode.OFF -> "QuickSetFlash_is_off"
+                        CameraFlashMode.AUTO -> "QuickSetFlash_is_auto"
+                        CameraFlashMode.ON -> "QuickSetFlash_is_on"
```

Comment:
lets rename these.

"QUICK_SETTING_FLASH_OFF" etc, so we can easily see what the tags refer to specifically